### ISSUE TITLE
[Bug]: xAxis & yAxis prop deprecation warning

### ISSUE THREAD

**Issue by statusunknown418** [tremorlabs/tremor-npm#1024]
### Tremor Version

3.16.1

### Link to minimal reproduction

https://stackblitz.com/edit/stackblitz-starters-rbutqp?description=The%20React%20framework%20for%20production&file=app/page.tsx,tailwind.config.ts&title=Next.js%20Starter

### Steps to reproduce

1. open the preview in another window
2. see the error appear in the console

### What is expected?

I guess the error should be more descriptive or not appear at all

### What is actually happening?

throwing a console error

**Comment by severinlandolt** [comment]
Thanks! Will take a look at updating Recharts. Will report back, thanks!

**Comment by severinlandolt** [comment]
Hi @statusunknown418, could you please test, thanks!

```
npm i @tremor/react@3.16.2-beta-version-bump.1
```

**Comment by severinlandolt** [comment]
c.f. https://github.com/recharts/recharts/issues/3615#issuecomment-2018091669

**Comment by statusunknown418** [comment]
cool, looking forward for the new release 🙏

**Comment by xdev00** [comment]
Its still  bugging?

**Comment by severinlandolt** [comment]
Yes, the solution is currently on an recharts alpha branch. We don't update to alpha branches at Tremor, hence the message still pops up.

**Comment by jakevollkommer** [comment]
> Yes, the solution is currently on an recharts alpha branch. We don't update to alpha branches at Tremor, hence the message still pops up.

Can we suppress the warning somehow in the meantime?

**Comment by ckifer** [comment]
This was resolved on the recharts side a while ago and from the version in the tremor package.json this should be resolvable/resolved
